.test-infra/terraform/google-cloud-platform/google-kubernetes-engine/.beam-utility.apache-beam-testing.tfbackend

```diff
@@ -0,0 +1,20 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+bucket = "b507e468-52e9-4e72-83e5-ecbf563eda12"
+prefix = ".test-infra/terraform/google-cloud-platform/google-kubernetes-engine/beam-utility"
```

Comment:
After changing the name of the cluster, could you also change this prefix to match?